### ISSUE TITLE
[server] Remove the HTTP server (like in Bookari)

### ISSUE THREAD

**Issue by jmgeffroy** [Mantano/iridium#42]
Removing the HTTP server has a lot of positive consequences:
- The contents is no longer accessible to other apps
- Requests responses are no longer serialized, and this provides a huge boost in performances (99% of the time is spent in this serialization, according to recent profiling).

https://pub.dev/documentation/flutter_inappwebview/latest/flutter_inappwebview/WebView/onLoadResourceCustomScheme.html

`onLoadResourceCustomScheme` is an `InAppWebView` cross-platform option, so it should be OK on both Android and iOS...

Note: See in Bookari (Mantano private repo)
-  `EpubWebViewClient.shouldInterceptRequest` 
- `Epub3WebViewFragment.mustRunServer`

See also:
- iOS: https://github.com/pichillilorenzo/flutter_inappwebview/issues/675#issuecomment-776187909
- Android-specific shouldInterceptRequest in https://pub.dev/packages/flutter_inappwebview/example

**Comment by jmgeffroy** [comment]
First tests are disappointing from a performance point of view. The reason is that "you shouldn't do any heavy processing inside shouldInterceptRequest, as it is a synchronous call (albeit on another thread). Calls for all network requests are processed on the same thread so blocking for one request will cause all of the other requests to be blocked as well" ([source](https://stackoverflow.com/questions/30180529/webviewclient-shouldinterceptrequest-freezes-the-webview)).
So we should spawn isolates.

**Comment by jmgeffroy** [comment]
Putting this on hold, and implementing isolates in the server version

**Comment by westernbuptboy** [comment]
note and bookmark not work on Android 6 because paginationInfo and currentSpineItemContext is null. related to this issue?

**Comment by gwangbaekun** [comment]
Yes. related. Bloc doesn't add data anymore. You should manually set paginationInfo with the position.
